TTS/tts/models/base_tts.py

```diff
@@ -287,7 +287,7 @@ def get_data_loader(
             loader = None
         else:
             # setup multi-speaker attributes
-            if hasattr(self, "speaker_manager") and self.speaker_manager is not None:
+            if self.speaker_manager is not None:
```

Comment:
Or is there something that I am missing? Do you have any specific reason why this will be safe to do?

Reply:
BaseTTS itself seems to set it on the constructor, so afaik it should always exist? (unless the subclass constructor doesn't call the superclass constructor)

Reply:
Oh Yeah fair! That makes sense. Since each model inherits the BaseTTS to use the tokenizer and other components it should be fine then.